src/core/lib/iomgr/udp_server.h

```diff
@@ -58,12 +58,11 @@ typedef void (*grpc_udp_server_orphan_cb)(grpc_exec_ctx *exec_ctx,
                                           grpc_fd *emfd);
 
 /* Create a server, initially not bound to any ports */
-grpc_udp_server *grpc_udp_server_create(void);
+grpc_udp_server *grpc_udp_server_create(struct grpc_server *server);
```

Comment:
I'd like to expunge grpc_server from udp_server entirely (what if I want a UDP socket without a grpc_server -- the use case for which is the reason this code is still in this repository)... I didn't realize it was already there, and I'd like to see it expunged quickly. 

It looks like it's there solely so it can be passed up in a read callback. A `void* user_data` field would be more appropriate there.

As for this usage, I think passing down channel args explicitly is entirely appropriate. You'll probably want to make a copy of them during construction (which is something we plan on making cheap... around EOQ3).